reporter/reporter.go

```diff
@@ -57,10 +58,13 @@ func New(
 		btccParamsRes *btcctypes.QueryParamsResponse
 		err           error
 	)
-	err = retry.Do(retrySleepTime, maxRetrySleepTime, func() error {
+	err = retrywrap.Do(func() error {
 		btccParamsRes, err = babylonClient.BTCCheckpointParams()
 		return err
-	})
+	},
+		retry.Delay(retrySleepTime),
```

Comment:
I would say that, given `max-attempts` is in common potion of the config it should probably set in every program, otherwise we risk seeing some wierd behaviour only to learn some toold respect this config option and some does not

Reply:
Agreed added it in [f7c96e3](https://github.com/babylonlabs-io/vigilante/pull/29/commits/f7c96e39b7bc9ae631eee54b16d44d2e889fba60)